blocks/library/cover-image/test/index.js

```diff
@@ -5,7 +5,14 @@ import { name, settings } from '../';
 import { blockEditRender } from 'blocks/test/helpers';
 
 jest.mock( 'blocks/media-upload', () => () => '*** Mock(Media upload button) ***' );
-
+jest.mock(
+	'@wordpress/data',
+	() => ( {
+		withSelect() {
```

Comment:
This might become hard to maintain. I hope we will find a better way to mock all those HOCs that would work in other cases, too. Latest posts block is another example, where we might have a similar issue.

Reply:
yes, I totally agree this may become a problem. I think an option would be for the very used HOCs like withSelect to create a testing util that allows simple predefined mocks, to their behavior.

Reply:
Yeah, it makes sense what you said. We need to observe it and performed some refactoring as soon as it becomes hard to maintain 👍